data/types/device/status/status.go

```diff
@@ -95,6 +97,12 @@ func (s *Status) Validate(validator structure.Validator) {
 	}
 
 	validator.Int("duration", s.Duration).GreaterThanOrEqualTo(DurationMinimum) // TODO: .Exists() - Suspend events on Animas do not have duration?
+	expectedDurationValidator := validator.Int("expectedDuration", s.DurationExpected)
```

Comment:
naming flips ... `expectedDuration`  => `DurationExpected`

Reply:
Check out `combination.go`. This is consistent with the rest of the code. The JSON field name is `expected*` while the Go type name is the reversed `*Expected`. This is on purpose because for V2 I'll be renaming all of the JSON fields to follow the Go type name format.

For constants, types, variable names, etc. I try to use the name and then the qualifier. So:

Duration
DurationExpected
DurationMaximum
DurationMinimum

Much easier to visually parse then finding all MaximumDuration among all of the other Maximum* names.